From cd14e0b13d7a2f92cdeb68bc367dc78fc88adb56 Mon Sep 17 00:00:00 2001
From: Steve Longerbeam <steve_longerbeam@mentor.com>
Date: Mon, 16 Mar 2015 16:12:15 -0700
Subject: [PATCH 631/668] media: imx6 camera: adv7180: Set MPU_STIM_INTRQ bit

Analog Devices recommends setting the MPU_STIM_INTRQ bit in
the INT_CONFIG_1 register on the ADV7180.

Port of linux-meibp 16d95c5b11dfef3fdb45c77c2cf83eb868073f44.

Signed-off-by: Steve Longerbeam <steve_longerbeam@mentor.com>
---
 drivers/staging/media/imx6/capture/adv7180.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/media/imx6/capture/adv7180.c b/drivers/staging/media/imx6/capture/adv7180.c
index 1a62c4c..8b57d93 100644
--- a/drivers/staging/media/imx6/capture/adv7180.c
+++ b/drivers/staging/media/imx6/capture/adv7180.c
@@ -1012,7 +1012,7 @@ static int adv7180_enable_interrupts(struct adv7180_dev *sensor)
 	/* Switch to interrupt register map */
 	ADV7180_WRITE_REG(sensor, ADV7180_ADI_CTL_1, 0x20);
 	/* INTRQ active low, active until cleared */
-	ADV7180_WRITE_REG(sensor, ADV7180_INT_CONFIG_1, 0xd1);
+	ADV7180_WRITE_REG(sensor, ADV7180_INT_CONFIG_1, 0xd5);
 	/* unmask SD_UNLOCK and SD_LOCK */
 	ADV7180_WRITE_REG(sensor, ADV7180_INT_MASK_1,
 			  ADV7180_INT_SD_UNLOCK | ADV7180_INT_SD_LOCK);
-- 
2.4.10

